app/scripts/controllers/preferences.js

```diff
@@ -111,6 +111,9 @@ export default class PreferencesController {
       ///: END:ONLY_INCLUDE_IF
       isLineaMainnetReleased: false,
       useExternalNameSources: true,
+      ///: BEGIN:ONLY_INCLUDE_IF(transaction-simulation)
+      useSimulations: true,
```

Comment:
Just for future proofing, should we be specific and call this `useTransactionSimulations`?